app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: int,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, car: Car) -> None:
+        income = 0.0
+        for i in car:
+
+            if i.clean_mark < self.clean_power:
+                total = self.calculate_washing_price(i)
+                self.wash_single_car(i)
+                income += total
+        return round(income, 1)
```

Comment:
u don't "for" cycle to process one single car
+
"i" is a bad name for counter variable
+
remove "for" cycle or change typehint car: Car